lib/Parser/RdfXml.php

```diff
@@ -94,7 +94,6 @@ protected function initXMLParser()
             xml_set_element_handler($parser, [$this, 'startElementHandler'], [$this, 'endElementHandler']);
             xml_set_character_data_handler($parser, [$this, 'cdataHandler']);
             xml_set_start_namespace_decl_handler($parser, [$this, 'newNamespaceHandler']);
-            xml_set_object($parser, $this);
```

Comment:
Just for reference: https://github.com/sweetrdf/easyrdf/pull/50/commits/aa98fe930c933a4e2ca3b8ec99258627806c21fb